src/backends/gr.jl

```diff
@@ -824,7 +824,7 @@ function gr_display(sp::Subplot{GRBackend}, w, h, viewport_canvas)
             # create the colorbar of contour levels
             if sp[:colorbar] != :none
                 gr_set_viewport_cmap(sp)
-                l = round(Int32, 1000 + (h - minimum(h)) / (maximum(h) - minimum(h)) * 255)
+                l = round(Int32, 1000 + (h - NaNMath.minimum(h)) / (NaNMath.maximum(h) - NaNMath.minimum(h)) * 255)
```

Comment:
I checked the GR backend code and I don't see any reason why not use the NaN-compliant functions. You can safely replace them. There is another call in line 642:

```julia
rmin, rmax = GR.adjustrange(minimum(r), maximum(r))
```